cookbooks/fb_users/resources/default.rb

```diff
@@ -69,15 +68,11 @@
 
   # Now we can add all the users
   node['fb_users']['users'].each do |username, info|
-    if info['action'] == :delete
-      user username do # ~FB014
-        action :remove
-      end
-      next
-    end
+    # helper variables
     mapinfo = ::FB::Users::UID_MAP[username]
     pgroup = info['gid'] || node['fb_users']['user_defaults']['gid']
     homedir = info['home'] || "/home/#{username}"
+    homedir_group = info['homedir_group'] || pgroup
```

Comment:
I believe pgroup is a gid, but homedir_group is a group, you'll need to translate that one.

Reply:
I don't think `pgroup` is a gid at this point. It's translated later in the user resource at line 119